src/data/Endpoint.js

```diff
@@ -46,6 +46,7 @@ const endpoints = {
     OPTIMIZE_ROUTE: `${laravelUrlPrefix}/${version}/customer/route/optimization`,
     ROUTE_LOCATION: `${laravelUrlPrefix}/${version}/customer/route/{0}/route-location`,
     DRIVER_SCHEDULE: `${laravelUrlPrefix}/${version}/customer/driver-schedule/{0}`,
+    DRIVER_UPDATE: `${laravelUrlPrefix}/${version}/customer/driver/{0}`,
```

Comment:
Shouldn't this be the same endpoint as +    CUSTOMER_DRIVER: `${laravelUrlPrefix}/${version}/customer/driver`,
Only a different method? @vdanylov

Reply:
@maartenhemmes according documentation, no.
<img width="560" alt="screen shot 2018-05-11 at 11 49 29" src="https://user-images.githubusercontent.com/20551843/39918754-bfca602e-5511-11e8-85f8-1d8c1977a80f.png">